examples/next/pages/index.tsx

```diff
@@ -55,20 +55,20 @@ const Home = (): JSX.Element => {
                     <Box>
                         <Title1>Using links</Title1>
                         <Stack space={16}>
-                            <Text>
```

Comment:
for some reason, using the Text component produced hydration warnings

Reply:
This is something I noticed while working on fixing the fixed footer SSR errors. There is an issue already: https://jira.tid.es/browse/WEB-1430